test/io-tests.sh

```diff
@@ -207,15 +202,17 @@ ensureAppSettings(){
 }
 
 checkAppSettingsReload(){
-  pgrStart "./configs/sigusr2-settings.config"
+  configFile=$(mktemp)
+  trap "rm -f $configFile" ERR RETURN
+  cat "./configs/sigusr2-settings.config" > "$configFile"
+  pgrStart "$configFile"
```

Comment:
This is a fun one... `postgrest-watch postgrest-test-io` was running in an infinite loop before, because those tests change the config file on disk, which results in a restart, which... you know! :)

I copied the config to a tempfile and then made the changes there. Outside of our source dir -> no problemo!